libappstream-glib/as-icon.c

```diff
@@ -513,11 +513,12 @@ as_icon_node_insert_embedded (AsIcon *icon, GNode *parent, AsNodeContext *ctx)
 	n = as_node_insert (parent, "icon", NULL, 0,
 			    "type", as_icon_kind_to_string (priv->kind),
 			    NULL);
-	if (as_node_context_get_version (ctx) >= 0.8) {
+	if (as_node_context_version_at_least (ctx, "0.8")) {
 		as_node_add_attribute_as_uint (n, "width", priv->width);
 		as_node_add_attribute_as_uint (n, "height", priv->height);
 	}
-	if (priv->scale > 1 && as_node_context_get_version (ctx) > 0.9)
+	// XXX - figure out real minimum version for "scale"
```

Comment:
Maybe @ximion  knows?

Reply:
The minimal `scale` is 1, the maximum is undefined (but currently nothing gets bigger than 2x). I wanted to point you at the spec here, but it looks like this feature was implemented in AppStream without documenting it. Eww. (I'll fix that later)

Reply:
I am looking for the version of the AppStream spec that introduced the "scale" attribute for icons of type "embedded".  I couldn't quickly find that icon type in the spec, is it maybe not yet specified at all?

Reply:
`embedded` is a proprietary extension that is AFAIK only used by OpenSUSE. I will never ever allow it in the spec.

Reply:
Okay, how do we control whether appstream-builder outputs `embedded`?  Should the version enum have a AS_FORMAT_EXPERIMENTAL value?